job-dsls/jobs/kie_docs_pr.groovy

```diff
@@ -26,7 +26,7 @@ mavenGoals = "-B clean install"
 PrVerificationJob.addFolders(this)
 
 // Creates or updates a free style job.
-def jobDefinition = job(Constants.PULL_REQUEST_FOLDER + "/${projectName}-pullrequests")
+def jobDefinition = job(Constants.PULL_REQUEST_FOLDER + "/${projectName}" + "-pullrequests-" + Constants.BRANCH)
```

Comment:
This ``" + "`` should not be necessary to add.

Reply:
But not a blocker for merging, this branch is also already in maintenance mode, if I understand properly.